### DISCUSSION TITLE
Note about the output aspect ratio

### DISCUSSION THREAD

**Discussion by Paper99** [TencentARC/PhotoMaker#121] [General]
Provided by [@DiscoNova](https://github.com/DiscoNova).

The underlying generative model has billions of parameters. And, whether You believe it or not, **all** of them do affect the output. The width/height (i.e. the "Aspect Ratio") of the output are just a few of these parameters.

For example, using the exact same positive/negative prompts and seed value, only changing the aspect ratio (i.e. the width/height) of the output image, it is possible to generate e.g. following results:

<p align="center">
  <img src="https://cdn.cleverest.eu/attachment/github.com/TencentARC/PhotoMaker/pull/120/aspect-ratio-matters.png" height=725>
</p>

**Comment by dusterio** [comment]
Maybe it's worth exposing width and height parameters in the cog.yaml then?

**Comment by Citramones** [comment]
Perhaps you can help me. I use the standard code given in the example from "How to Test" with custom Lora models for SDXL 1.0 and a basic SDXL model from PhotoMaker. When generating an image, styles do not always work. and this may depend on adapter_weights but how to configure it correctly? and another problem is that the face has no common features with the reference
